ipod-shuffle-4g.py

```diff
@@ -423,18 +440,23 @@ def construct(self, tracks):
 
         # Build all the remaining playlists
         playlistcount = 1
+        podcastlistcount = 0
         for i in self.lists:
             playlist = Playlist(self)
             verboseprint("[+] Adding playlist", (i[0] if type(i) == type(()) else i))
             playlist.populate(i)
             construction = playlist.construct(tracks)
             if playlist["number_of_songs"] > 0:
+                if playlist["listtype"] == 3:
```

Comment:
Can we get an enum for that?

```
1 is the master playlist
2 is a normal playlist
3 is a podcast playlist
4 is a audiobook playlist
```

See https://github.com/nims11/IPod-Shuffle-4g/blob/master/docs/iTunesSD3gen.md#playlistx

Reply:
I actually thought about that and figured it'd be simpler to just use the constants. The Enum is pretty straightforward

Reply:
Can you add that please?

Reply:
I just need to push the changes